.dialyzer_ignore.exs

```diff
@@ -0,0 +1,3 @@
+[
+  {"lib/open_telemetry/span.ex", :invalid_contract}
```

Comment:
technically all of the span macros are invalid as they return AST. I think this is the only way to solve it unless we want to remove the typespecs

Reply:
Hm, why is that invalid? Isn't that how they should work?

Reply:
Technically a macro is always returning AST, so dialyzer will not accept that we annotate it with a spec that says it returns a boolean

Reply:
Oh, duh :)